platform/jupyter/Dockerfile

```diff
@@ -0,0 +1,6 @@
+From python:3.11
```

Comment:
@rstarmer can you rebase and remove this PR from the change?

Reply:
should be gone